strip-comments.js

```diff
@@ -8,8 +10,10 @@ var CommentStripper = (function (window) {
     var NEW_LINE = '\n';
     var CARRIAGE_RETURN = '\r';
 
-    var CommentStripper = function () {
-
+    var CommentStripper = function (options) {
```

Comment:
I don't love the options object, especially when there's only one option.  i'd prefer either just have `preserveNewLines` as `public`, or getter and setter methods for it.  of course the readme would need to be updated as well